### PR TITLE
dbus-daemon: add page

### LINKED ISSUE #937: page request: dbus-daemon
dbus-daemon man page request

### PR DESCRIPTION
**Pull request by pxgamer**
Closes #937

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/dbus-daemon.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Run the daemon with the standard per-login-session message bus configuration:
+
+`dbus-daemon --session`
+
+- Run the daemon with the standard systemwide message bus configuration:
+
+`dbus-daemon --system`
+
+- Set the address to listen on and ignore the configuration for this:
```
Comment (agnivade) [inline pages/linux/dbus-daemon.md]: What does "this" mean here ? Do you mean the configuration file ?

File: pages/linux/dbus-daemon.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Output the process id to stdout:
+
+`dbus-daemon --print-pid`
+
+- Force the message bus to write to the system log for messages:
+
+`dbus-daemon --syslog`
+
+- Disable the creation of a pid file and ignore the configuration for this:
```
Comment (agnivade) [inline pages/linux/dbus-daemon.md]: Is this a common enough use-case ? I believe usually you would want to use a pid file ? (Also, same question as above)

Comment (pxgamer) [reply pages/linux/dbus-daemon.md]: I guess probably not really. 🤔

File: pages/linux/dbus-daemon.md (hunk 1 above)
Comment (pxgamer) [reply pages/linux/dbus-daemon.md]: "this" is referring to the configuration setting inside the config file. There should be a value in the file such as `<listen>unix:path=/tmp/foo</listen>`. I guess maybe `and override any value in the configuration file`? Or something like that.

Comment (agnivade) [reply pages/linux/dbus-daemon.md]: Ah ok, I think it maybe be better written as - `Set the address to listen on and override the configuration value for it.`

File: pages/linux/dbus-daemon.md (hunk 2 above)
Comment (pxgamer) [reply pages/linux/dbus-daemon.md]: Sure, I'll remove it when I get to work. :+1:

**Pull request merged**